### PR TITLE
Feature/ajv

### PR DESCRIPTION
**Pull request by niktrix**

### CODE REVIEW

File: src/helpers/validate.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "$id": "/properties/address",
+      "type": "string",
+      "title": "The Address Schema ",
+      "default": "",
+      "examples": [
+        "0x0000000000000"
+      ]
+    }
+  }
+};
```
Comment (aldoborrero) [inline src/helpers/validate.ts]: Add space

File: src/helpers/validate.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const AddressValidator = ajv.compile(addressSchema);
+export {AddressValidator}
```
Comment (aldoborrero) [inline src/helpers/validate.ts]: Add space

File: src/helpers/validate.ts [hunk 3]
```diff
@@ -1,22 +1,37 @@
...
+      type: 'string',
+      enum: ["BEGIN","YEAR","MONTH","DAY"],
+      title: 'The Duration Schema ',
+      default: '',
     }
   }
-};
-const AddressValidator = ajv.compile(addressSchema);
-export {AddressValidator}
+}
```
Comment (aldoborrero) [inline src/helpers/validate.ts]: Add space

File: src/helpers/validate.spec.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai'
+import { AddressValidator, DurationValidator, TokensValidator } from './validate'
+
+describe('validate', () => {
+  describe('AddressValidator() method', () => {
+    const msg = { address: '0xDECAF9CD2367cdbb726E904cD6397eDFcAe6068D' }
+    const wrongmessage = { address: '0xDECAF9CD2367cdbb726E904cD6397eDFcAe6068D' }
+    it('should return true', () => {
```
Comment (aldoborrero) [inline src/helpers/validate.spec.ts]: Add spaces (remember to properly align code vertically and horizontally)

File: tsconfig.json [hunk 5]
```diff
@@ -18,6 +18,7 @@
     }
   },
   "include": [
+    ".",
```
[the remark this answers is not in the document]
Comment (niktrix) [reply tsconfig.json]: For tsconfig.json

File: src/helpers/validate.spec.ts (hunk 4 above)
Comment (niktrix) [reply src/helpers/validate.spec.ts]: done

**Comment by niktrix** [comment]
Closing this as changes are already merged

**Pull request closed without merging**